scripts/launch/node.sh

```diff
@@ -53,11 +57,23 @@ witness_params="
 --bind 127.0.0.1:4010:4011
 --db-path $tmp_files/witness.db
 --log-dir $tmp_files/logs
---witness-listen 127.0.0.1:4020
+--witness-listen $witness_web_addr
 --witness-keyfile $tmp_files/witness.key
 --witness-gen-key
 "
 
+# parameters for faucet
+faucet_params="
```

Comment:
It's better to write a separate `faucet.sh` script that will be used on the deployment.

Reply:
Lol, I couldn't get what's wrong with it